### DISCUSSION TITLE
Zoom shouldn't be the only meeting provider

### DISCUSSION THREAD

**Discussion by echoboomer** [incidentbot/incidentbot#60] [Ideas]
Make it more flexible for which provider is used for meetings and meeting links.

**Comment by echoboomer** [comment]
Done.

**Comment by chadcancode** [comment]
How can I enable Teams?
Even Zoom... what do I enter in the config such that when a new incident is kicked off, a new call is created and a link to that video call is posted at the top of the incident Slack channel?

**Reply by echoboomer** [reply]
@chadcancode The bot doesn't currently generate a link for anything, it simply pastes in the value of the meeting link provided as part of the config to remain platform agnostic. If there is interest in integrating further with specific providers, I wouldn't mind adding that as a feature if the minimum set of providers could be agreed upon.

**Reply by echoboomer** [reply]
The auto Zoom feature has been implemented.
